<!DOCTYPE html>
<html lang="en">
<title>W3.CSS Template</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<style>
body,h1,h2,h3,h4,h5,h6 {font-family: "Lato", sans-serif}
.w3-bar,h1,button {font-family: "Montserrat", sans-serif}
.fa-anchor,.fa-coffee {font-size:200px}
</style>
<body>

<!-- Navbar -->
<div class="w3-top">
  <div class="w3-bar w3-blue w3-card w3-left-align w3-large">
    <a class="w3-bar-item w3-button w3-hide-medium w3-hide-large w3-right w3-padding-large w3-hover-white w3-large w3-red" href="javascript:void(0);" onclick="myFunction()" title="Toggle Navigation Menu"><i class="fa fa-bars"></i></a>
    <a href="#" class="w3-bar-item w3-button w3-padding-large w3-white">Top</a>
    <a href="https://doi.org/10.1101/2020.10.16.343210" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white">Pre-print</a>
    <a href="https://github.com/shainalu/spatial_rep" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white">Code</a>
    <a href="https://molecularatlas.org" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white">ST Data</a>
    <a href="#Contact" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white">Contact</a>
  </div>

  <!-- Navbar on small screens -->
  <div id="navDemo" class="w3-bar-block w3-white w3-hide w3-hide-large w3-hide-medium w3-large">
    <a href="https://doi.org/10.1101/2020.10.16.343210" class="w3-bar-item w3-button w3-padding-large">Pre-print</a>
    <a href="https://github.com/shainalu/spatial_rep" class="w3-bar-item w3-button w3-padding-large">Code</a>
    <a href="https://molecularatlas.org" class="w3-bar-item w3-button w3-padding-large">ST Data</a>
    <a href="#Contact" class="w3-bar-item w3-button w3-padding-large">Contact</a>
  </div>
</div>

<!-- Header -->
<header class="w3-container w3-blue w3-center" style="padding:80px 32px">
  <h1 class="w3-margin w3-xxxlarge">Does spatially resolved gene expression capture Allen Brain Atlas labels and does this replicate across datasets? </h1>
  <p class="w3-xlarge">Shaina Lu*, Cantin Ortiz, Daniel Furth, Stephan Fischer, Dinos Meletis, Tony Zador, Jesse Gillis</h4>
  <p class="w3-large">*presenting author </p>
  <p class="w3-large">Cold Spring Harbor Laboratory and Karolinska Institutet </p>
  <!--<button class="w3-button w3-black w3-padding-large w3-large w3-margin-top">blah</button>-->
  <a href="https://doi.org/10.1101/2020.10.16.343210" class="w3-button w3-black w3-padding-large w3-large w3-margin-top">Read the pre-print</a>
</header>

<!-- background -->
<!--<div class="w3-row-padding w3-light-grey w3-padding-32 w3-container vertical-center">-->
<div class="w3-container w3-black w3-opacity w3-padding-64">
  <div class="w3-content vertical-center">
    <h2> Background: </h2>
    <p class="w3-text-white w3-large">Spatial gene expression is particularly interesting in the mammalian brain, with the potential to serve as a link between many data types. However, as with any type of expression data, cross-dataset benchmarking of spatial data is a crucial first step. Here, we assess the replicability, with reference to canonical brain sub-divisions, between the Allen Institute's in situ hybridization data from the adult mouse brain (ABA) and a similar dataset collected using Spatial Transcriptomics (ST). With the advent of tractable spatial techniques, for the first time we are able to benchmark the Allen Institute's whole-brain, whole-transcriptome spatial expression dataset with a second independent dataset that similarly spans the whole brain and transcriptome. </p>
</div>
</div>

<!-- First Grid; Figure 2 -->
<div class="w3-row-padding w3-padding-64 w3-container">
  <div class="w3-content">
    <div class="w3-third">
      <h3>Canonical brain areas are classifiable using gene expression alone in the ABA and ST datasets</h3>
      <p class="w3-text-grey"> Heat map of AUROC for classifying leaf brain areas from all other leaf brain areas in (A) ABA and (B) ST using LASSO (alpha = 0.1). Dendrograms on the far left side represent clustering of leaf brain areas based on the inverse of AUROC; areas with an AUROC near 0.5 get clustered together while areas with an AUROC near 1 are further apart. Color bar on the left represents the major brain structure that the leaf brain area is grouped under. These areas include: cortex (CTX), midbrain (MB), cerebellum (CB), striatum and pallidum (CNU), hindbrain (HB), and thalamus and hypothalamus (IB). </p>

    </div>

    <div class="w3-twothird">
      <img src="fig2ab.png" alt="fig2a,b" class="w3-image" width=100% height=auto>
      <!-- image source for github: /spatial_rep/fig2ab.png -->
    </div>
  </div>
</div>

<hr>

<!-- Second Grid; Figure 3 -->
<div class="w3-row-padding w3-padding-64 w3-container">
  <div class="w3-content">

    <div class="w3-third">
      <img src="fig3abc.png" alt="fig2a,b" class="w3-image" width=100% height=auto>
    </div>

    <div class="w3-twothird">
      <h3 class="w3-margin-left">Training in ST generalizes to ABA, but the reverse is not true</h3>
      <p class="w3-text-grey w3-margin-left"> Models trained with overlapping genes and brain areas between ST and ABA datasets are evaluated within dataset on the test fold and across dataset on the entire opposite dataset. (A) Summary diagrams showing mean AUROC for within dataset test setperformance (purple arrow) and cross dataset performance with models trained in the opposite dataset (light blue arrow) for LASSO (alpha = 0.1). (B and C) Distributions of AUROCs for within (purple) and cross dataset (light blue) performance for (B) LASSO (alpha = 0.1) trained in ST and (C) LASSO (alpha = 0.1) trained in ABA. Dashed vertical lines represent the mean of the corresponding colored distribution. </p>

    </div>

  </div>
</div>

<hr>

<!-- Third Grid; Figure 4 -->
<div class="w3-row-padding w3-padding-64 w3-container">
  <div class="w3-content">
    <div class="w3-third w3-margin-right">
      <h3> Cross-dataset performance is partially explained by distance in the phylogeny of brain areas of the reference atlas, but not by Euclidean distance in the brain</h3>
      <p class="w3-text-grey"> Cross-dataset AUROCs (x-axis) of classifying all leaf brain areas from all other leaf brain areas for (A) ABA using LASSO (alpha = 0.1) trained in ST and (B) ST using LASSO (alpha = 0.1) trained in ABA as a function of path length (x-axis) in the ARA naming hierarchy between the two brain areas being classified. The same AUROCs (y-axis) from (A) and (B) shown in (C) and (D) respectively as a function of minimum Euclidean distance between the two brain areas in the ARA (x-axis). Euclidean distance on the x-axis is binned into deciles for visualization. All four plots show mean AUROCs (points) with standard deviation (vertical bars). </p>

    </div>

    <div class="w3-rest w3-margin-left">
      <img src="fig4abcd.png" alt="fig2a,b" class="w3-image" width=100% height=auto>
      <!-- image source for github: /spatial_rep/fig2ab.png -->
    </div>
  </div>
</div>

<hr>

<!-- Fourth Grid; Figure 5 -->
<div class="w3-row-padding w3-padding-64 w3-container">
  <div class="w3-content">

    <div class="w3-twothird">
      <img src="fig5ab.png" alt="fig5a,b" class="w3-image" width=100% height=auto>
    </div>

    <div class="w3-third">
      <h3 class="w3-margin-left">Leaf brain area expression profiles are identifiable within dataset, but do not
generalize cross dataset</h3>
      <p class="w3-text-grey w3-margin-left"> (A) Schematic depicting training and testing schema. Models are trained to classify one leaf brain area against the rest of the brain (one v. all) and then used to test classification of that brain area against all other leaf brain areas (one v. one) within and across dataset. (B) Performance similarity between one v. all and one v. one reported as the mean absolute value of distance from identity line for scatter plots of testing in one v. all against one v. one. Performance similarity shown for within ST, ST to ABA, within ABA, and ABA to ST across linear regression (red), LASSO (alpha = 0.01) (green), and LASSO (alpha = 0.05) (violet). </p>

    </div>

  </div>
</div>

<hr>

<!-- Conclusions -->
<div class="w3-container w3-black w3-opacity w3-padding-64">
  <div class="w3-content vertical-center">
    <h2> Results: </h2>
    <p class="w3-text-white"> We use LASSO, linear regression, and correlation-based feature selection in a supervised learning framework to classify expression samples relative to their assayed location. We show that Allen reference atlas labels are classifiable using transcription, but that performance is higher in the ABA than ST. Further, models trained in one dataset and tested in the opposite dataset do not reproduce classification performance bi-directionally. Finally, while an identifying expression profile can be found for a given brain area, it does not generalize to the opposite dataset.</p>
    <h2> Conclusions: </h2>
    <p class="w3-text-white"> In general, we found that canonical brain area labels are classifiable in gene expression space within dataset and that our observed performance is not merely reflecting physical distance in the brain. However, we also show that cross-platform classification is not robust. Emerging spatial datasets from the mouse brain will allow further characterization of cross-dataset replicability.</p>
  </div>
</div>

<!-- Footer -->
<footer class="w3-container w3-padding-32 w3-center w3-black w3-opacity" id="Contact"> 
  <p class="w3-xlarge">Contact: <a href="mailto:slu@cshl.edu">slu@cshl.edu</a></p>
  <div class="w3-xlarge w3-padding-16">
    <a href="mailto:slu@cshl.edu"> <i class="fa fa-envelope w3-hover-opacity"></i></a>
    <a href="https://github.com/shainalu/spatial_rep"> <i class="fa fa-github w3-hover-opacity"></i></a>
    <a href="https://twitter.com/starfishlu"> <i class="fa fa-twitter w3-hover-opacity"></i></a>
    <a href="https://www.linkedin.com/in/shainalu/"> <i class="fa fa-linkedin w3-hover-opacity"></i></a>
 </div>
 <!--<p>Powered by <a href="https://www.w3schools.com/w3css/default.asp" target="_blank">w3.css</a></p>-->
</footer>

<script>
// Used to toggle the menu on small screens when clicking on the menu button
function myFunction() {
  var x = document.getElementById("navDemo");
  if (x.className.indexOf("w3-show") == -1) {
    x.className += " w3-show";
  } else { 
    x.className = x.className.replace(" w3-show", "");
  }
}
</script>

</body>
</html>


